math/check_amicable_pair.cpp

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright 2020 @author iamnambiar
+ * 
+ * @file
+ * A C++ Program to check whether a pair of number is amicable pair or not.
```

Comment:
```suggestion
 * \brief A C++ Program to check whether a pair of number is [amicable pair](https://en.wikipedia.org/wiki/Amicable_numbers) or not.
```
You can use the GitHub [Markdown](https://guides.github.com/features/mastering-markdown/) and HTML tags to write the documentation.